assets/airports/othh.json

```diff
@@ -321,7 +321,7 @@
       "name": "Afnan",
       "suffix": {"16L":"1M", "16R":"1C", "34L":"1W", "34R":"1E"},
       "entryPoints": {
-        "AFNAN": [["AFNAN", "A120-"]]
+        "AFNAN": [["KUPSA"], ["AFNAN", "A120-"]]
```

Comment:
The array here isn't necessary; it can just be a string. I'm not sure if a query is done on the type when each portion of the route is sent various places, but it may use a type comparison string/array to determine whether it is a fix or a fix-restriction-pair.